doc/ranger.pod

```diff
@@ -205,6 +205,11 @@ This feature relies on the dimensions of the terminal's font.  By default, a
 width of 8 and height of 11 are used.  To use other values, set the options
 C<iterm2_font_width> and C<iterm2_font_height> to the desired values.
 
+=head3 terminology
+
+This only works in terminology. It can render vector graphics, but works only locally.
+To enable this feature, set the option C<preview_images_method> to terminology.
```

Comment:
Grammar nazi patrol : )
Missing new line before this one. Don't actually care about whether there's one or not but we should keep it consistent.

Reply:
All other sections have this line in a separate paragraph.